### PR TITLE
Output cpu time instead of cpu busy percentage

### PR DESCRIPTION
**Pull request by yayaha**
CPU time can directly be computed as cpu_busy_pct * total_time. Since it's a duration, it'll fit into the current time-to-k8s charts better than cpu_busy_pct.

Sample output on my personal laptop:

|name        |args                                                        |platform|iteration|time                                                  |version                  |exitcode|error|command exec (seconds)|apiserver answering (seconds)|kubernetes svc (seconds)|dns svc (seconds)|app running (seconds)|dns answering (seconds)|cpu time (seconds)|total duration (seconds)|
|------------|------------------------------------------------------------|--------|---------|------------------------------------------------------|-------------------------|--------|-----|----------------------|-----------------------------|------------------------|-----------------|---------------------|-----------------------|------------------|------------------------|
|minikube-114|minikube start --driver=docker --kubernetes-version=v1.14.10|darwin  |1        |2021-09-04 17:17:19.178595 -0700 PDT m=+624.864202022 |minikube version: v1.22.0|       |     |70.435                |1.797                        |0.068                   |0.069            |6.658                |0.170                  |15.998            |79.198                  |
|minikube-115|minikube start --driver=docker --kubernetes-version=v1.15.12|darwin  |1        |2021-09-04 17:18:44.219993 -0700 PDT m=+709.905430551 |minikube version: v1.22.0|       |     |70.254                |1.712                        |0.069                   |0.068            |7.821                |0.174                  |16.281            |80.098                  |
|minikube-116|minikube start --driver=docker --kubernetes-version=v1.16.3 |darwin  |1        |2021-09-04 17:20:10.31169 -0700 PDT m=+795.996956542  |minikube version: v1.22.0|       |     |70.557                |1.866                        |0.106                   |0.110            |7.635                |25.366                 |18.810            |105.641                 |
|minikube-117|minikube start --driver=docker --kubernetes-version=v1.17.3 |darwin  |1        |2021-09-04 17:22:02.242473 -0700 PDT m=+907.927516542 |minikube version: v1.22.0|       |     |60.227                |1.528                        |0.064                   |0.058            |18.127               |5.169                  |18.038            |85.173                  |
|minikube-118|minikube start --driver=docker --kubernetes-version=v1.18.3 |darwin  |1        |2021-09-04 17:23:33.331963 -0700 PDT m=+999.016825324 |minikube version: v1.22.0|       |     |58.779                |1.556                        |0.065                   |0.059            |18.794               |0.168                  |17.475            |79.422                  |
|minikube-119|minikube start --driver=docker --kubernetes-version=v1.19.0 |darwin  |1        |2021-09-04 17:24:58.859762 -0700 PDT m=+1084.544454003|minikube version: v1.22.0|       |     |58.889                |1.640                        |0.065                   |0.064            |14.606               |0.199                  |17.079            |75.463                  |

### CODE REVIEW

**Comment by tstromberg** [comment]
Can you include both CPU busy % and CPU time? At least for myself, the percentage is a lot easier to reason about.

**Comment by yayaha** [comment]
Done.

|name        |args                                                        |platform|iteration|time                                                  |version                  |exitcode|error|command exec (seconds)|apiserver answering (seconds)|kubernetes svc (seconds)|dns svc (seconds)|app running (seconds)|dns answering (seconds)|cpu time (seconds)|total duration (seconds)|cpu busy (percent)|
|------------|------------------------------------------------------------|--------|---------|------------------------------------------------------|-------------------------|--------|-----|----------------------|-----------------------------|------------------------|-----------------|---------------------|-----------------------|------------------|------------------------|------------------|
|minikube-117|minikube start --driver=docker --kubernetes-version=v1.17.3 |darwin  |1        |2021-09-13 10:25:40.293523 -0700 PDT m=+558.729117410 |minikube version: v1.22.0|       |     |55.387                |1.572                        |0.078                   |0.073            |17.563               |35.506                 |16.582            |110.179                 |15.05             |
|minikube-118|minikube start --driver=docker --kubernetes-version=v1.18.3 |darwin  |1        |2021-09-13 10:27:36.55062 -0700 PDT m=+674.985990977  |minikube version: v1.22.0|       |     |51.735                |1.760                        |0.070                   |0.072            |21.523               |0.162                  |14.314            |75.322                  |19.00             |
|minikube-119|minikube start --driver=docker --kubernetes-version=v1.19.0 |darwin  |1        |2021-09-13 10:28:58.261012 -0700 PDT m=+756.696225360 |minikube version: v1.22.0|       |     |50.989                |1.691                        |0.070                   |0.069            |19.246               |25.340                 |15.539            |97.405                  |15.95             |
|minikube-114|minikube start --driver=docker --kubernetes-version=v1.14.10|darwin  |1        |2021-09-13 10:30:41.73958 -0700 PDT m=+860.174593178  |minikube version: v1.22.0|       |     |66.027                |1.532                        |0.065                   |0.058            |7.996                |25.325                 |15.219            |101.003                 |15.07             |
|minikube-115|minikube start --driver=docker --kubernetes-version=v1.15.12|darwin  |1        |2021-09-13 10:32:28.957386 -0700 PDT m=+967.392192497 |minikube version: v1.22.0|       |     |70.104                |1.479                        |0.065                   |0.060            |6.979                |0.149                  |13.752            |78.837                  |17.44             |
|minikube-116|minikube start --driver=docker --kubernetes-version=v1.16.3 |darwin  |1        |2021-09-13 10:33:53.698001 -0700 PDT m=+1052.132644864|minikube version: v1.22.0|       |     |66.216                |1.641                        |0.067                   |0.056            |8.459                |0.150                  |13.916            |76.590                  |18.17             |

File: time-to-k8s.go [hunk 1]
```diff
@@ -273,7 +276,7 @@ func main() {
 
 	c := csv.NewWriter(outputFile)
 
-	c.Write([]string{"name", "args", "platform", "iteration", "time", "version", "exitcode", "error", "command exec (seconds)", "apiserver answering (seconds)", "kubernetes svc (seconds)", "dns svc (seconds)", "app running (seconds)", "dns answering (seconds)", "cpu busy (percent)", "total duration (seconds)"})
```
Comment (tstromberg) [inline time-to-k8s.go:276]: Can you preserve the previous column order for "cpu busy (percent)"? I don't want to break anyones existing workflow.

Comment (yayaha) [reply time-to-k8s.go:276]: Done.

**Pull request merged**